public/resources/stylesheets/userbar.less

```diff
@@ -477,7 +477,7 @@ a.navbar-login:hover {
 
       li {
         display: block;
-        padding: 12px;
+        padding: 0px;
```

Comment:
I've fixed the resulting code as per your requested change. If there's anything else let me know